app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -0,0 +1,98 @@
+package org.fossasia.openevent.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.fossasia.openevent.Adapters.SessionsAdapter;
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.data.Speaker;
+import org.fossasia.openevent.dbutils.DbSingleton;
+import org.fossasia.openevent.utils.SpeakerIntent;
+
+import java.text.ParseException;
+import java.util.List;
+
+/**
+ * Created by MananWason on 30-06-2015.
+ */
+public class SpeakersActivity extends AppCompatActivity {
+    SessionsAdapter sessionsAdapter;
+    private String speaker;
+    private Speaker clicked;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_speakers);
+        DbSingleton dbSingleton = DbSingleton.getInstance();
+        speaker = getIntent().getStringExtra("SPEAKER");
+        final Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar_speakers);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        try {
+            clicked = dbSingleton.getSpeakerbySpeakersname(speaker);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+
+        TextView biography = (TextView) findViewById(R.id.speaker_bio);
+        ImageView linkedin = (ImageView) findViewById(R.id.imageView_linkedin);
+        ImageView twitter = (ImageView) findViewById(R.id.imageView_twitter);
+        ImageView github = (ImageView) findViewById(R.id.imageView_github);
+        ImageView fb = (ImageView) findViewById(R.id.imageView_fb);
+
+        biography.setText(clicked.getBio());
+        final SpeakerIntent speakerIntent = new SpeakerIntent(clicked, getApplicationContext());
+
+        speakerIntent.mIntent(github);
```

Comment:
what is `mIntent` ?? if it is a function it should not start with `mIntent` as its name.